src/main/java/nl/ordina/jobcrawler/service/ScraperService.java

```diff
@@ -58,10 +92,46 @@ public void scrape() {
                 Optional<Vacancy> existCheck = vacancyService.findByURL(vacancy.getVacancyURL());
                 if (existCheck.isPresent()) {
                     existVacancy++;
+                    allVacancies.remove(vacancy);
                 } else {
-                    Set<Skill> skills = skillMatcherService.findMatchingSkills(vacancy);
-                    vacancy.setSkills(skills);
-                    vacancyService.save(vacancy);
+                    String vacancyLocation = vacancy.getLocationString();
+                    if(vacancyLocation.endsWith(", Nederland")) {vacancyLocation = vacancyLocation.substring(0,vacancyLocation.length()-11);}
```

Comment:
Use multiple lines instead of using one line for this if statement.
Curly brackets aren't necessary.

Other solution is to use the ternary operator, although that line might be a bit long.
```java
vacancyLocation = vacancyLocation.endsWith(", Nederland") ? vacancyLocation.substring(0,vacancyLocation.length()-11) : vacancyLocation;
```